tools/stats/upload_stats_lib.py

```diff
@@ -266,3 +275,46 @@ def get_job_id(report: Path) -> int | None:
         return int(report.parts[0].rpartition("_")[2])
     except ValueError:
         return None
+
+
+@lru_cache
+def get_job_name(
+    id: int | None, workflow_id: int | None, workflow_run_attempt: int | None
+) -> str | None:
+    if id is None:
+        return None
+    try:
+        if workflow_id is None:
```

Comment:
Could you remind me on why `workflow_id` can be `None` here as this should be available from https://github.com/pytorch/pytorch/blob/main/.github/workflows/upload-test-stats.yml#L96?